ltm/agent.py

```diff
@@ -302,21 +305,17 @@ def reply(self, user_content: str, cost_callback: Callable[[float], Any] = None)
         return response
 
     def _truncated_completion(self, context: LLMContext, max_messages: int, **kwargs) -> str:
-        while len(context) + 1 > max_messages or self.count_tokens(messages=context) > self.max_prompt_size:
+        while len(context) + 1 > max_messages or self.count_tokens(messages=context) > self.max_input_tokens:
             context.pop(1)
         return self._completion(context, **kwargs)
 
     def _completion(self, context: List[dict[str, str]], temperature: float, label: str,
                     cost_callback: Callable[[float], Any]) -> str:
-        # TODO: max_tokens is borked: https://github.com/BerriAI/litellm/issues/4439
-        response = completion(model=self.model, messages=context, timeout=self.config.timeout,
-                              temperature=temperature)
-        response_text = response['choices'][0]['message']['content']
+        response_text = ask_llm(
+            context, self.model, temperature=temperature, timeout=self.config.timeout, cost_callback=cost_callback,
+        )
```

Comment:
We can add `max_response_tokens` back in here since they fixed this in the new litellm. Might be worth it to remove the version requirement on litellm